controller/repliesController.js

```diff
@@ -132,20 +132,23 @@ const createReply = async (req, res, next) => {
   }
 };
 
-const getReplyVotes = async (req, res, next) => {
-  try {
-    const { commentId, replyId } = req.params;
-    const { voteType } = req.query;
-
-    if (!ObjectId.isValid(commentId)) {
-      return next(new CustomError(404, "Invalid ID"));
-    }
-
-    if (!ObjectId.isValid(replyId)) {
-      return next(new CustomError(404, "Invalid ID"));
-    }
-
-    const parentComment = await Comments.findById(commentId);
+exports.upVoteReply = async (req, res, next) => {
+try {
+        const { commentId, replyId } = req.params;
+        const { voteType } = req.query;
+        const { ownerId} = req.body;
+
+        if (!ObjectId.isValid(commentId)) {
+          return next(new CustomError(404, "Invalid ID"));
+          }
+
+        if (!ObjectId.isValid(replyId)) {
+          return next(new CustomError(404, "Invalid ID"));
+          }
+      
+
+    const parentComment = await Comments.findById({ _id: commentId
+    });
 
     // Check to see if the parent comment exists.
     if (!parentComment) {
```

Comment:
What's happening here? I think that you are missing some code. If you tested your code then this would have been flagged.

Reply:
Got it. Will work on these